Misc/NEWS.d/next/Library/2021-11-11-13-03-17.bpo-45235.8ZbkHa.rst

```diff
@@ -0,0 +1,3 @@
+Reverted an argparse bugfix that caused regression in the handling of
+default arguments for subparsers.  This prevented leaf level arguments from
```

Comment:
The word "This" seems a little bit ambiguous to me. It can mean this PR (#29525) or the bugfix/regression.